api/web3server_utils.go

```diff
@@ -117,13 +117,16 @@ func (svr *web3Handler) getTransactionFromActionInfo(blkHash hash.Hash256, selp
 	if err != nil {
 		return nil, err
 	}
+	tx, err := action.RawTxToSignedTx(ethTx, svr.coreService.ChainID(), selp.Signature())
```

Comment:
tx.WithSignature(...) should be called inside ToEthTx(), which returns an unsigned tx incorrectly, instead?

Reply:
`ToEthTx()` is designed to return the raw/unsigned tx, which is passed to `rlpRawHash()` to compute the raw hash (to be signed), at this moment there's no signature yet